src/javascripts/_helpers.js

```diff
@@ -91,12 +91,32 @@ var UK_Parliament = (function () {
 
       throw 'Sorry, there is no such ancestor with the identifiers you referenced: ' + JSON.stringify(attributes);
 
+    },
+
+    /**
+     * Toggle class to switch styles
+     * @param {object} element Element to toggle class on
+     * @param {string} active_state By default, this is set to 'active'
+     */
+    toggleClass = function(element, active_state) {
+
+      active_state = active_state || 'active';
```

Comment:
Is this logical operator required?

Reply:
I think it's useful because then the user won't have to set an active state if by default they would like the active class to be 'active', which feels like a name that can be used for anything. This just reduces extra typing when the user needs to declare the function.

This: `UK_Parliament.toggleClass(element, 'active');`

Can be this instead: `UK_Parliament.toggleClass(element);`

Unless you would like to make the active class name to be different:
`UK_Parliament.toggleClass(element, 'open');`